src/Marten/Schema/SQL/mt_stream.sql

```diff
@@ -39,7 +39,7 @@ DECLARE
 	seq int;
 	return_value int[];
 BEGIN
-	select version into event_version from {databaseSchema}.mt_streams where id = stream;
+	select version into event_version from {databaseSchema}.mt_streams where id = stream for update;

```

Comment:
I think the SQL should not have `for update` clause, request other reviewers to validate and confirm on this.

Reply:
I went ahead and remove it from there